src/main/resources/library/attribute-definitions.json

```diff
@@ -231,9 +231,16 @@
     "library:lmsvn": {
       "title": "Library Metadata Schema Version Number",
       "description": "Library Metadata Schema Version Number",
-      "type": "integer",
-      "default": 3,
-      "indexable": false
+      "current": 3,
+      "indexable": false,
+      "type": "integer"
+    },
+    "library:dulvn": {
+      "title": "Structured Data Use Limitations  Version Number",
+      "description": "Structured Data Use Limitations Version Number",
+      "current": 1,
```

Comment:
I was thinking that a default value is one you change only if it hasn't been set before, whereas we change to the current version number every time you catalog.  But I'm fine with calling it default.